### PR TITLE
Auto rest

### PR DESCRIPTION
**Pull request by calexan**
Adding initial version of sdk for Azure RemoteApp

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@calexan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (calebal). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci test this please

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by calexan** [comment]
I've disabled non-passing tests for now and opened a bug to track them for fixing in the next few weeks. Builds successfully on my local machine with command "msbuild build.proj /t:build;package;test /p:scope=ResourceManagement\RemoteApp"

File: src/ResourceManagement/RemoteApp/Microsoft.Azure.Management.RemoteApp/project.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+  "version": "0.0.28-preview",
```
Comment (markcowl) [inline src/ResourceManagement/RemoteApp/Microsoft.Azure.Management.RemoteApp/project.json]: this should match the assemblyfileversion in assemblyInfo.cs

File: src/ResourceManagement/RemoteApp/Microsoft.Azure.Management.RemoteApp/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+//      Major Version
+//      Minor Version 
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers 
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("0.9.0.0*")]
+[assembly: AssemblyVersion("0.9.0.0")]
+[assembly: AssemblyFileVersion("0.0.25.0")]
```
Comment (markcowl) [inline src/ResourceManagement/RemoteApp/Microsoft.Azure.Management.RemoteApp/Properties/AssemblyInfo.cs]: this should match the version in project.json

File: src/ResourceManagement/RemoteApp/RemoteApp.Tests/RemoteAppTestBase.cs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.Rest.ClientRuntime.Azure.TestFramework;
+using RemoteApp.Tests;
+
+namespace Microsoft.Azure.Management.RemoteApp.Tests
+{
+    public class RemoteAppTestBase
+    {
+        protected RemoteAppManagementClient GetClient(MockContext context, RemoteAppDelegatingHandler handler)
+        {
+            //RemoteAppManagementClient client = context.GetServiceClient<RemoteAppManagementClient>(handler);
```
Comment (markcowl) [inline src/ResourceManagement/RemoteApp/RemoteApp.Tests/RemoteAppTestBase.cs]: This code should not be commented out.

File: global.json [hunk 4]
```diff
@@ -10,7 +10,8 @@
     "src/ResourceManagement/Graph.RBAC/Microsoft.Azure.Graph.RBAC",
     "src/ResourceManagement/DataLake.Analytics/Microsoft.Azure.Management.DataLake.Analytics",
     "src/ResourceManagement/DataLake.Store/Microsoft.Azure.Management.DataLake.Store",
-    "src/ResourceManagement/DataLake.StoreUploader/Microsoft.Azure.Management.DataLake.StoreUploader"
+    "src/ResourceManagement/DataLake.StoreUploader/Microsoft.Azure.Management.DataLake.StoreUploader",
+    "src/ResourceManagement/DataLake.StoreUploader/Microsoft.Azure.Management.RemoteApp"
```
Comment (markcowl) [inline global.json]: shouldn't this be src/ResourceManagement/RemoteApp/Microsoft.Azure.Management.RemoteApp?

**Pull request merged**